docker.go

```diff
@@ -101,3 +101,37 @@ func loadDockerImage(cli *client.Client, pathToImage string) (string, error) {
 
 	return string(b[:]), nil
 }
+
+// Import the specified image into docker with repotag
+// Returns the message produced by the docker daemon
+func importDockerImage(cli *client.Client, pathToImage string, repoTag string) (string, error) {
+	image, err := os.Open(pathToImage)
+	if err != nil {
+		return "", err
+	}
+	defer image.Close()
+
+	// ImageImport require as attributes special structures
+	// ImageImportSource and ImageImportOptions
+	// Thye are defined in docker/api/types/client.go
+	var image_src types.ImageImportSource
+	var image_opt types.ImageImportOptions
+
+	// Source is the data to send to the server to create this image from
+	image_src.Source = image
+	// SourceName is the name of the image to pull.
+	// Set to "-" to leverage the Source attribute
+	image_src.SourceName = "-"
+	repotag := strings.Split(repoTag, ":")
+	image_opt.Tag = repotag[1]
```

Comment:
In Go the convention is to name variables in lowerCamelCase. Moreover, I don't know if it's guaranteed that repoTag is always like this "image:tag" (e.g. maybe it's just "repo" while assuming it's "latest"). So, with some simplifications and in a more Go style, I'd write it like this:

```go
var tag string
repotag := strings.Split(repoTag, ":")
if len(repotag) == 2 {
  tag = repotag[1]
} else {
  tag = "latest"
}

imageSrc := types.ImageImportSource{ Source: image }
imageOpt := types.ImageImportOptions{ Tag: tag }
```